src/ripple/basics/impl/contract.cpp

```diff
@@ -39,6 +41,18 @@ accessViolation() noexcept
 
 } // detail
 
+void
+LogThrow (std::string title)
+{
+    if (auto j = debugJournal().debug())
+    {
+        j << title;
+        for (auto const& frame : beast::getStackBacktrace ())
+            j << "    " << frame;
+        j << "----";
+    }
+}
```

Comment:
Woah.  I'm sure this works, but it looks expensive to me since we're logging a backtrace of every `Throw` and `Rethrow` regardless of whether the `Throw` is unhanded at the top of the thread.

A cheaper solution (at runtime) might be to add similar logging in a `catch(...)` in the top function of every new thread.  That solution would also log unhanded exceptions coming from places like the standard library or soci, not just our own exceptions.  It would be cheaper because it would only log the unhanded exceptions -- not all of them.  Just my thoughts on the matter.